PowerEditor/installer/nativeLang/croatian.xml

```diff
@@ -955,13 +955,38 @@
 			<Preference title="Postavke">
 				<Item id="6001" name="Zatvori"/>
 				<Global title="Općenito">
+					<Item id="6123" name="Lokalizacija:"/>
+					<Item id="6131" name="Glavni izbornik"/>
+					<Item id="6122" name="Sakrij (za prikaz koristite Alt ili F10)"/>
+					<Item id="6132" name="Sakrij prečace ＋ ▼ ✕ s desne strane"/>
+					<Item id="6133" name="Statusna vrpca"/>
+					<Item id="6134" name="Sakrij"/>
+				</Global>
+
+				<Toolbar title="Alatna vrpca">
 					<Item id="6102" name="Sakrij"/>
 					<Item id="6103" name="Fluent UI ikone: male"/>
 					<Item id="6104" name="Fluent UI ikone: velike"/>
 					<Item id="6129" name="Fluent UI ikone: ispunjene male"/>
 					<Item id="6130" name="Fluent UI ikone: ispunjene velike"/>
 					<Item id="6105" name="Klasične ikone: male"/>
-
+					<Item id="6011" name="Kolorizacija"/>
+					<Item id="6012" name="Potpuno"/>
+					<Item id="6013" name="Djelomično"/>
+					<Item id="6014" name="Izbor boje"/>
+					<Item id="6015" name="Podrazumijevano"/>
+					<Item id="6016" name="Crveno"/>
+					<Item id="6017" name="Plavo"/>
+					<Item id="6018" name="Zeleno"/>
```

Comment:
Take care of interchanged ids 6017 and 6018, see https://github.com/notepad-plus-plus/notepad-plus-plus/commit/48d953693ad9bd2829f1dfd36c852504f8dc4a6f

Reply:
+1